### DISCUSSION TITLE
HDD attributes in card

### DISCUSSION THREAD

**Discussion by PrzemekSkw** [kalkih/mini-graph-card#1168] [General]
Hello, it's possible to show all attributes in mini-graph-card?
![Screenshot_2024-11-20-23-09-08-57_c3a231c25ed346e59462e84656a70e50](https://github.com/user-attachments/assets/07800031-2781-4b01-9979-99377a96cc33)
Regards.

**Comment by ildar170975** [comment]
All attributes with numerical values can be shown as graphs by using an `attribute` option.
Check Docs.

**Reply by PrzemekSkw** [reply]
Hi, thanks for Your answer. I ask question here because it's very unclear for me:
![Screenshot_2024-11-20-23-46-49-33_e4424258c8b8649f6e67d283a50a2cbc](https://github.com/user-attachments/assets/174f55bd-93cd-45d2-8aab-44a90a31498c)

I don't understand how can I setup my drive attributes that method.
Regards.

**Comment by ildar170975** [comment]
This part of Docs is related to attributes stored as dictionaries.
Not your case.
Suggest you to ask similar questions in the main dedicated thread of Community: https://community.home-assistant.io/t/lovelace-mini-graph-card/71385

**Reply by ildar170975** [reply]
As a starting point: this code for showing a graph for attribute `yyy` of `sensor.xxx`:
```
type: custom:mini-graph-card
entities:
  - entity: sensor.xxx
    attribute: yyy
```

**Reply by PrzemekSkw** [reply]
Hi, I try that but don't work.
![Uploading Screenshot_2024-11-21-07-12-42-91_c3a231c25ed346e59462e84656a70e50.jpg…]()
